### ISSUE TITLE
Experimental instalation (loading plugins from plugin folder) not working

### ISSUE THREAD

**Issue by lfilho** [alexanderjeurissen/ranger_devicons#47]
Hi there! Thanks a lot for your efforts on this! Appreciate it!

I have set it up using the recent support for plugins according to the [current instructions](https://github.com/alexanderjeurissen/ranger_devicons/tree/e64e1da1f1b8f3aa23989507f78e1bc3f48944da#experimental-method). The only difference was that instead of cloning the git repo, I added it as submodule:

```
git submodule add https://github.com/alexanderjeurissen/ranger_devicons ranger/plugins/ranger_devicons

ls ~/.config/ranger/plugins/ranger_devicons
LICENSE          LICENSE_NERDFONT Makefile         README.md        __init__.py      devicons.py      screenshot.png
``` 
Then after this, i have started `ranger` but no icons. Then I started a new shell session and tried again, also no luck. Then I entered the plugin above and run `make install` then it worked. I then generated new files on `ranger/plugins`:

```
ls ~/.config/ranger/plugins/
__init__.py           __init__.pyo          devicons.py           devicons.pyo          devicons_linemode.py  devicons_linemode.pyo ranger_devicons
```
So now I have to either add these files above to `.gitignore` or keep committing them everytime.

Wasn't it supposed to work without the manual `make install` step? Am I missing something?

Thank you!

**Comment by alexanderjeurissen** [comment]
Depending on what machine you are running this, the ranger version that you have installed might not support the `experimental plugin installation` yet. For instance last time I checked on OSX, the Homebrew version of ranger does not include the commit that adds support to ranger for this way of installing plugins.

see: https://github.com/alexanderjeurissen/ranger_devicons/pull/37

which is the original PR that adds this functionality, it refers to a ranger upstream commit that is required for this to work. my hunch is that your ranger version is either out of date or that the latest version available in your package manager of choice does not include that commit yet.

For now `make install` is the stable way of installing this extension and it works reliably.

**Comment by lfilho** [comment]
Indeed! I just checked and although i have the lastest released version (1.9.2) that version is much behind their master. I wonder why they don't release (at least minor) versions more often. I'll close this one here then. Cheers!
